### PR TITLE
docs: Fix several typos in the angular documentation

### PR DESCRIPTION
**Pull request by ghost**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by alan-agius4 — APPROVED** [review]
Thanks for this. LGTM

File: aio/content/guide/template-typecheck.md [hunk 1]
```diff
@@ -348,7 +348,7 @@ Care should be taken that if an `ngAcceptInputType_` override is present for a g
 Disable checking of a binding expression by surrounding the expression in a call to the [`$any()` cast pseudo-function](guide/template-expression-operators).
 The compiler treats it as a cast to the `any` type just like in TypeScript when a `<any>` or `as any` cast is used.
 
-In the following example, casting `person` to the `any` type suppresses the error `Property address does not exist`.
+In the following example, casting `person` to the `any` type suppresses the error `Property addresss does not exist`.
```
Comment (gkalpak) [inline aio/content/guide/template-typecheck.md:351]: This needs reverting :smiley:

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx, @Michael-Doner :100:

One small comment - otherwise lgtm :+1:

File: aio/content/guide/template-typecheck.md (hunk 1 above)
Comment (alan-agius4) [reply aio/content/guide/template-typecheck.md:351]: Actually this is correct as the error would be `Property addresss does not exist`

Comment (gkalpak) [reply aio/content/guide/template-typecheck.md:351]: Oh, you're right 🤦‍♂️
I didn't take the time to look at this more carefully 😇

**Review by gkalpak — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Review by jessicajaniuk — APPROVED** [review]
reviewed-for: public-api

**Comment by jessicajaniuk** [comment]
This does not merge cleanly with `13.3.x`. Please create a patch PR. Updating to `target: minor`.

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit f1630bb24d9b003fa27736ed67714f3da7cb01f7.

**Pull request closed without merging**